build/tools/Automation/DotNet/DotNet.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace Microsoft.Oryx.Automation
+{
+    /// <Summary>
+    ///
+    /// TODO:
+    ///     - Replace Console.WriteLine with Logging
+    ///     - Add unit tests
+    ///
+    /// This class is reponsible for encapsulating logic for automating SDK releases for DotNet.
+    /// This includes:
+    ///     - Getting new release version and sha
+    ///     - Updating constants.yaml with version and sha
+    ///         - This is important so build/generateConstants.sh
+    ///           can be invoked to distribute updated version
+    ///           throughout Oryx source code. Which updates
+    ///           Oryx tests.
+    ///     - Updating versionsToBuild.txt
+    /// </Summary>
+    public class DotNet : Program
+    {
+        /// <Summary>
+        /// Gets DotNet's new release version and sha.
+        ///
+        /// This is accomplished by:
+        ///     - Checking release meta data url if there's a new release
+        ///     - If new release, then store release information into PlatformConstants
+        ///        Otherwise don't store anything
+        /// </Summary>
+        /// <returns>PlatformConstants used later to update constants.yaml</returns>
+        public override async Task<List<PlatformConstant>> GetPlatformConstantsAsync(string dateTarget)
+        {
+
+            // check dotnet releases' meta data
+            var response = await Request.RequestAsync(Constants.DotNetReleasesMetaDataUrl);
+            var json = JsonConvert.DeserializeObject<ReleaseNotes>(response);
```

Comment:
I'd rename this variable to `releaseNotes` since it's not a JSON string but the deserialization of the JSON string into the `ReleaseNotes` object

Reply:
Updated